src/cockpit/packages.py

```diff
@@ -149,7 +149,7 @@ class Package:
     PO_JS_RE: ClassVar[Pattern] = re.compile(r'po\.([^.]+)\.js(\.gz)?')
 
     # A built in base set of "English" translations
-    PO_EN_JS: ClassVar[Entity] = b'', mimetypes.guess_type('po.js')
```

Comment:
What's the story with this change?  Is that invalid syntax in 3.6?

Reply:
Yes, it is:
```
Traceback (most recent call last):
  File "/usr/lib64/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib64/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/src/py36/src/cockpit/bridge.py", line 36, in <module>
    from .packages import Packages, PackagesListener
  File "/src/py36/src/cockpit/packages.py", line 152
    PO_EN_JS: ClassVar[Entity] = b'', mimetypes.guess_type('po.js')
                                    ^
SyntaxError: invalid syntax
```

Also, I really wish that would continue to be the case in recent Python. I stared at this line for several minutes before I finally figured out what it's supposed to mean. The grouping of `=` and `,` is really very unhappy here, it wasn't clear at all that this is a tuple assignment.